### ISSUE TITLE
How to build a devbundle for Raspberry Pi

### ISSUE THREAD

**Issue by cronicryo** [meteor/meteor#7230]
Currently ive been trying to build a dev bundle for a raspberry pi.  (yes there are ones that are built but theyre not as up to date with meteor)

My specific problem is that it fails installing all the packages. The only conclusion i can come to is im not using the right node version.
Ive tried 0.10.43 which is in the current generate file
Ive tried the latest (6.2.1) which interestingly enough fails further down when installing the packages

So long story short please provide us with the requirements to build the bundle with a little more detail in what versions to use

**Comment by laosb** [comment]
I'm not professional in generating a dev bundle, but I think using a modified `[generate-dev-bundle.sh](https://github.com/meteor/meteor/blob/devel/scripts/generate-dev-bundle.sh)` should do this trick.

Btw, I'm not sure what version of Raspberry Pi you're using, but it can be something about the architecture you're using.

**Comment by laosb** [comment]
This should be posted to Meteor Forums instead of issues. Closing.

**Comment by Lewiscowles1986** [comment]
Can I ask why an issue belongs in the forums and not on GitHub?

**Comment by abernix** [comment]
@Lewiscowles1986 Issues are reserved for documented bugs in Meteor with reproductions (either a repository or explicit instructions) in order to track them until they are resolved.  What was reported above was not a bug and seemed to have enough uncertainty that it sounded more like an issue with knowing how to properly compile the dependencies that Meteor needed.
